src/seismometer/controls/selection.py

```diff
@@ -205,39 +231,50 @@ def __init__(
         if value is not None:
             values[value[0]] = value[1]
         else:
-            value = (list(values.keys())[0], values[list(values.keys())[0]])
+            value = (tuple(values.keys())[0], values[tuple(values.keys())[0]])
```

Comment:
value initializing seems very convoluted - note to self to investigate further

Reply:
added some comments, and re-wrote the code to hopefully make this more transparent.